### PR TITLE
WRO-12570: Update .travis.yml to apply lint options

### PR DESCRIPTION
**Pull request by sjro**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Need to apply lint option ( strict, max-warnings , report-unused-disable-directives) when building a PR.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

Update .travis.yml to apply lint options

### Links
[//]: # (Related issues, references)
WRO-12570

### Comments
Enact-DCO-1.0-Signed-off-by: SJ RO (sj.ro@lge.com)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ script:
     - npm run test
     - echo -e "\x1b\x5b35;1m*** Tests complete\x1b\x5b0m"
     - echo -e "\x1b\x5b35;1m*** Starting eslint...\x1b\x5b0m"
-    - npm run lint
+    - npm run lint -- -- --report-unused-disable-directives --max-warnings 0 .
     - cd ../../..
-    - npm run lint
+    - npm run lint -- --report-unused-disable-directives --max-warnings 0 .
```
Comment (MikyungKim) [inline .travis.yml]: Is the last `.` needed?

Comment (sjro) [reply .travis.yml]: Yes. If you write a specific file in the location where dot is, lint is performed only on that file.
Dot seems to mean all files under the current directory recursively.

Comment (MikyungKim) [reply .travis.yml]: I meant `npm run lint` already has `eslint .` so I thought this may be redundant.

Comment (sjro) [reply .travis.yml]: Removed unnecessary dots.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**